packages/textfield/src/textfield.css

```diff
@@ -134,5 +134,7 @@ textarea {
     left: 0;
     height: 100%;
     resize: none;
-    overflow: hidden;
+    white-space: pre-line;
+    min-height: max-content;
+    overflow: clip;
```

Comment:
Did you mean to maybe put this at line 54?

Reply:
I tried these css changes but they didn't work, neither on line 137 nor on line 54.
In both cases the textarea was not growing at all!

Reply:
There is a CSS answer for this. Please find it.

Reply:
Apparently `white-space: pre-line;`  is sufficient enough for this